### PR TITLE
Create pkgdown website

### LINKED ISSUE #336: Create rgbif pkgdown website
👋 @sckott I think it would be nice if rgbif has a [pkgdown](https://pkgdown.r-lib.org/) website, so users (including myself 😬) have a nice overview of the package, functions, vignettes and changelog. The [occurrence manual](https://ropensci.github.io/occurrence-manual/) (currently linked as the pkg website) doesn't cover that, but would be a good resource to reference from the website.

If you agree, I can set this up. We have done this before for https://ropensci.github.io/wateRinfo/ and http://adokter.github.io/bioRad/

### PR DESCRIPTION
**Pull request by peterdesmet**
## Description

Create a pkgdown website for `rgbif`. Preview of site: http://peterdesmet.github.io/rgbif/: it contains a [homepage](http://peterdesmet.github.io/rgbif/index.html), [overview of functions](http://peterdesmet.github.io/rgbif/reference/index.html), [vignettes](http://peterdesmet.github.io/rgbif/articles/index.html) and [changelog](http://peterdesmet.github.io/rgbif/news/index.html).

- Add a `_pkgdown.yml` file to configure site (f9cb762). Current configuration adds a Twitter handle for @sckott as author and uses the `yeti` theme (also used in `wateRinfo`), but many more options are possible.
- Build website in `/docs` using `pkgdown::build_site()` (9b0e6ea). Worked like a charm on the first go, which is a testament to the good documentation of this package! 👏 Site can be hosted using GitHub pages from `master/docs` (also the reason why this PR had to come from `master`).
- Add ORCIDs for contributors (found one for all except Laurens Geffert) in DESCRIPTION (8eed555). See preview at http://peterdesmet.github.io/rgbif/authors.html
- Replace occ-manual with URL of pkgdown website in DESCRIPTION (d43a34d)
- Move two images used in the README from `/tools` to `/man/figures` (where logo is located) so it can be found by `pkgdown`(386c5c5)
- Move code of conduct to `.github` where other GitHub-related files are also located.

## Todo if accepted

- Add yaml frontmatter to vignettes, to they [get a title](http://peterdesmet.github.io/rgbif/articles/index.html) (currently `UNKNOWN TITLE`) and ideally also author. @sckott I didn't want to do this without your feedback, but it is basically adding something [like this](https://github.com/ropensci/wateRinfo/blob/f84aa5ab4b83cb177f4dcc9a9edd7688573b62b9/vignettes/define_date_periods.Rmd#L1-L10) and removing the H1 title from the vignette. Let me know if that is OK.
- Update the links to the tutorials at the top of the README to their html page on the website, rather than the (less readable) .Rmd files. The webpage will link to the Rmd
- Configure repository to host GitHub pages website from `master/docs`
- Update repo website to `https://ropensci.github.io/rgbif`

## Related Issue

Fix #336

## Example

Site can be created by running:

```R
pkgdown::build_site()
```

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -11,17 +11,14 @@ Description: A programmatic interface to the Web Service methods
 Version: 1.1.0.9113
 License: MIT + file LICENSE
 Authors@R: c(
-    person("Scott", "Chamberlain", role = c("aut", "cre"),
-      email = "myrmecocystus@gmail.com",
-      comment = c(ORCID="0000-0003-1444-9135")),
-    person("Vijay", "Barve", role = "ctb"),
-    person("Dan", "Mcglinn", role = "ctb"),
-    person("Damiano", "Oldoni", role = "ctb"),
+    person("Scott", "Chamberlain", role = c("aut", "cre"), email = "myrmecocystus@gmail.com", comment = c(ORCID="0000-0003-1444-9135")),
```
Comment (sckott) [inline DESCRIPTION]: @peterdesmet let's add you as an author!

File: DESCRIPTION [hunk 2]
```diff
@@ -11,17 +11,14 @@ Description: A programmatic interface to the Web Service methods
...
+    person("Vijay", "Barve", role = "ctb", comment = c(ORCID = "0000-0002-4852-2567")),
+    person("Dan", "Mcglinn", role = "ctb", comment = c(ORCID = "0000-0003-2359-3526")),
+    person("Damiano", "Oldoni", role = "ctb", comment = c(ORCID = "0000-0003-3445-7562")),
     person("Laurens", "Geffert", role = "ctb"),
-    person("Karthik", "Ram", role = "ctb")
+    person("Karthik", "Ram", role = "ctb", comment = c(ORCID = "0000-0002-0233-1757"))
     )
-URL: https://github.com/ropensci/rgbif (devel) 
-    https://ropensci.github.io/occurrence-manual/ (manual)
+URL: https://github.com/ropensci/rgbif, https://ropensci.github.io/rgbif
```
Comment (sckott) [inline DESCRIPTION]: can we do all 3 urls? that is, something like: 

`https://github.com/ropensci/rgbif (devel)  https://ropensci.github.io/rgbif (docs)  https://ropensci.github.io/occurrence-manual/ (manual)`

Comment (peterdesmet) [reply DESCRIPTION]: 3 URLs are fine I think, but they should be comma separated (see http://r-pkgs.had.co.nz/description.html):

> Multiple URLs are separated with a comma.

Not sure about adding `(devel)` etc. Would keep it to URL only.

**Comment by peterdesmet** [comment]
> whereas here https://github.com/ropensci/wateRinfo/issues/new where you have the file in .github it doesn't show up

For me it does though: 🤔

screenshot 2019-01-21 at 10 49 08

File: DESCRIPTION (hunk 1 above)
Comment (sckott) [reply DESCRIPTION]: yeah, `ctb` is good. yeah, alphabetical sounds good.

File: DESCRIPTION (hunk 2 above)
Comment (sckott) [reply DESCRIPTION]: sounds good on the comma

**Comment by sckott** [comment]
ah okay, glad the links in a new issue shows up. disregard that comment

**Comment by peterdesmet** [comment]
- [x] Add to `.Rbuildignore
- [x] YAML frontmatter for vignettes + some spell check: 1a9efc1

**Comment by peterdesmet** [comment]
Note: the vignettes are now in order of their `%\VignetteIndexEntry{}`

title | filename | %\VignetteIndexEntry{}
--- | --- | ---
GBIF downloads | downloads.Rmd | Downloads
Cleaning data using GBIF issues | issues_vignette.Rmd | Cleaning data using GBIF issues
rgbif introduction | rgbif_vignette.Rmd | rgbif introduction
Taxonomic names | taxonomic_names.Rmd | Taxonomic names

I would suggest to update those to (in bold):

title | filename | %\VignetteIndexEntry{}
--- | --- | ---
**Downloading data from GBIF** | downloads.Rmd | **1_downloading**
Cleaning data using GBIF issues | **issues.Rmd** | **2_cleaning**
**Introduction to rgbif** | **rgbif.Rmd** | **0_introduction**
Taxonomic names | taxonomic_names.Rmd | **3_taxonomic_names**

Renaming `rgbif_vignette.Rmd` to `rgbif` will move it out of articles and make it a `Get started` page for the package in the navbar, which I think would be good. See e.g. https://adokter.github.io/bioRad/articles/bioRad.html

**Comment by sckott** [comment]
That sounds good to rename them.

**Comment by peterdesmet** [comment]
All done! I've also updated the section in the README that links to the tutorials (6f292e1), naming it "To get started, see" and referring to: introduction, function overview, articles and occ manual.

Once accepted, the only remaining things are to:

- Activate the website
- Update the repo website in description

**Comment by sckott** [comment]
perfect, thanks so much for the work on this @peterdesmet

**Pull request merged**